persistence/selecting.md

```diff
@@ -81,6 +82,12 @@ Since SQL Server and PostgreSQL have great support for querying JSON data, this
 
 SQL Persistence [allows accessing the `DbConnection` and `DbTransaction`](/persistence/sql/accessing-data.md) from within message handlers to enable saving business data and NServiceBus data within the same transaction.
 
+### MongoDB
+
+If a business system already stores its data in MongoDB, NServiceBus supports storing data inside MongoDB as well. This removes the need to introduce additional storage.
```

Comment:
This paragraph seems redundant. The equivalent statement can be made for any of the platforms, in which case the equivalent paragraph should be repeated in every section.